src/pudl/transform/ferc1.py

```diff
@@ -2366,14 +2366,6 @@ def apply_xbrl_calculation_fixes(
                     },
                 ],
                 "unappropriated_undistributed_subsidiary_earnings": [
-                    {
-                        "calc_component_to_replace": {},
-                        "calc_component_new": {
-                            "name": "unappropriated_undistributed_subsidiary_earnings_previous_year",
-                            "weight": 1.0,
-                            "source_tables": ["retained_earnings_ferc1"],
-                        },
-                    },
```

Comment:
Are these changes to the calculation fixes related to the dbf-only facts?

Reply:
ah no this is just one of the many instances of factoids that weren't actually in the data.